controllers/ovndbcluster_controller.go

```diff
@@ -612,6 +622,15 @@ func (r *OVNDBClusterReconciler) reconcileNormal(ctx context.Context, instance *
 
 		// Set DB Address
 		instance.Status.InternalDBAddress = strings.Join(internalDbAddress, ",")
+		if instance.Spec.DBType == ovnv1.SBDBType {
```

Comment:
can also check for networkattach != nil

Reply:
Done

Reply:
Not sure what the value is though. Even for ovnController handler (current state of the repo), it's not immediately clear why we generate only when attachments are present. (It doesn't seem to buy us anything?) I guess it's ok either way, I just don't see the need to introduce additional check when the worst that can happen is an unused ConfigMap. But I won't insist.